backend/services/interfaces/ILessonService.ts

```diff
@@ -25,10 +33,30 @@ export interface ILessonService {
   getLessonById(id: string): Promise<LessonResponseDTO>;
```

Comment:
I think this may be related to your previous comment [here](https://github.com/uwblueprint/rowan-house/pull/34#discussion_r795249898). But I think the same explanation applies here that if in the future we were to add more get Lesson methods, the specificity here would be beneficial.